data/categories.txt

```diff
@@ -2850,7 +2850,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
 en:street|st|str|ave|road|rd
 ru:улица|переулок|проезд|тракт|проспект|ул
 uk:вулиця|провулок|проїзд|проспект|бульвар|вул
-de:Strasse|Weg|Avenue|Gasse|Str
+de:Straße|Weg|Boulevard|Gasse
```

Comment:
Str was recently added by another german user. Are you sure it needs to be removed?